cypress/fixtures/develModeBlocks.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
I've created this fixture to simplify future testing of these features. 
It defines a standard set of blocks that should be state aware, and removes the need to define blocks in each test. 


Older tests have not yet been refactored to use it.

Reply:
Couldn't this sort of comment could go in the JSON itself, and maybe the filename could be more obvious?

Reply:
Done. Comment and file rename